src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js

```diff
@@ -84,7 +84,11 @@ class GmailLabelView {
       return;
     }
 
-    querySelector(this._element, '.av').style.maxWidth = maxWidth;
+    try {
+      querySelector(this._element, '.av').style.maxWidth = maxWidth;
+    } catch (e) {
+      // noop
+    }
```

Comment:
```suggestion
    this._element.querySelector('.av')?.style.maxWidth = maxWidth;
```

The querySelector function imported from querySelectorOrFail is specifically a function built to throw on failure as opposed to the standard querySelector method, so this code would be more natural to do with the standard querySelector method if we don't want that behavior.

Reply:
Wait, this is still a js file using Flow, and I don't think that supports optional chaining, unlike our Typescript files.

Reply:
This file shows off how hard it is to try converting Flow to TS piecemeal over years too. It's depended on by `gmail-thread-row-view` which has 5 flow files depending on it.

Reply:
```suggestion
    const childEl = this._element.querySelector('.av')
    if (childEl) {
      childEl.style.maxWidth = maxWidth;
    }
```

Reply:
This looks great, thanks for the suggestions.